### PR TITLE
Pyodide 0.25 Release Notes

### PR DESCRIPTION
**Pull request by ryanking13**
@rth @hoodmane

I added some topics that I can think of.

### CODE REVIEW

**Comment by hoodmane** [comment]
Thanks @ryanking13! I'll take a look soon.

**Comment by hoodmane** [comment]
Thanks @ryanking13! I'll take a look soon.

**Comment by hoodmane** [comment]
I guess we should add some docs here:
https://pyodide.org/en/latest/usage/api/js-api.html#pyodide.runPythonSyncifying
saying that it is:
1. unstable
2. won't work in runtimes without experimental flags
3. has unpolished i.e., buggy interactions with asyncio / the event loop and various other things

File: content/posts/0.25-release.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+await pyodide.loadPackage("requests")
+pyodide.runPython(`
+  import requests
+  license = requests.get("https://raw.githubusercontent.com/pyodide/pyodide/main/LICENSE").text
+  print(license)
+`)
+```
+
+Please note that `requests` in Pyodide are still subject to browser constraints,
+including CORS policies and limited access to low-level networking features not accessible from the browser.
```
Comment (hoodmane) [inline content/posts/0.25-release.md:56]: Maybe link here:
https://pyodide.org/en/latest/usage/wasm-constraints.html#synchronous-http-requests-support

File: content/posts/0.25-release.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+How does it work? [@joemarshall](https://github.com/joemarshall) has contributed a new adapter for `urllib3`, a backend of `requests`,
+that uses Web APIs supported by Pyodide instead of native sockets when running in a Pyodide environment.
+
+Try the following code snippet to see `requests` in action:
+
+```js
+await pyodide.loadPackage("requests")
+pyodide.runPython(`
+  import requests
+  license = requests.get("https://raw.githubusercontent.com/pyodide/pyodide/main/LICENSE").text
```
Comment (hoodmane) [inline content/posts/0.25-release.md:50]: for better readability:
```suggestion
  url = "https://raw.githubusercontent.com/pyodide/pyodide/main/LICENSE"
  license = requests.get(url).text
```

File: content/posts/0.25-release.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Happy New Year! Pyodide 0.25 is a major release with some exciting and experimental features.
+This major release includes experimental support for JS Promise integration, `requests` library support,
+and build system improvements.
+
+## Experimental Support for JS Promise Integration
+
+[TODO]
+
+## `requests` Support in Pyodide
```
[the remark this answers is not in the document]
Comment (ryanking13) [reply content/posts/0.25-release.md:28]: Make sense. Thanks!

**Comment by rth** [comment]
Thank you both for making this release happen !

**Comment by ryanking13** [comment]
Thanks for finishing the release notes!

**Pull request merged**